doc/observable.md

```diff
@@ -259,19 +259,19 @@ Conclusion:
 
 ## Anatomy of an Observable
 
-Observables are **created** using `Rx.Observable.create` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer an Subscription.
+Observables are **created** using `Rx.Observable.create` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer and Subscription.
 
 Four core Observable concerns:
-- **Creation**
+- **Create**
 - **Subscribe**
-- **Execution**
-- **Disposal**
+- **Execute**
+- **Dispose**
 
```

Comment:
It's not that simple. The `Subscription` represents the "Execution" of the Observable, but the `subscribe` is an important event where everything "starts" and gets set up. So it's not a matter of changing verbs to nouns.